Engine/Setup/BrokerageSetupHandler.cs

```diff
@@ -216,12 +216,12 @@ public bool Setup(SetupHandlerParameters parameters)
                         //Initialise the algorithm, get the required data:
                         algorithm.Initialize();
 
-                        if (algorithm.AccountCurrency != Currencies.USD)
-                        {
-                            throw new NotImplementedException(
-                                "BrokerageSetupHandler.Setup(): calling 'QCAlgorithm.SetAccountCurrency()' " +
-                                "is only supported in backtesting for now.");
-                        }
+                        //if (algorithm.AccountCurrency != Currencies.USD)
```

Comment:
Is this change intentional?

Reply:
Yeah. But might be supplied as separate PR. It's necessary for Binance as it uses USDT as account currency. But for now I've commented it out to attract additional attention :)

Reply:
Attention attracted indeed!

This requires broader support for non-USD account currencies to be properly implemented.

Reply:
Created a new issue #3593 that covers this problem

Reply:
@Adalyat I think we need broader non-USD support as a separate PR as a precursor to this one. Let's connect offline to discuss the work involved for that distinct PR.